test/t/calls/call_topics_test.exs

```diff
@@ -0,0 +1,19 @@
+defmodule T.CallTopicsTest do
+  use T.DataCase, async: true
+  use Oban.Testing, repo: T.Repo
+
+  alias T.CallTopics
+
+  describe "call topics" do
+    # TODO list of supported locales to ENVs?
+    test "call topics for every locale has equal size" do
+      locales = ["en", "ru"]
+
+      en_locale_length = length(CallTopics.locale_topics("en"))
+
+      for locale <- locales do
+        assert en_locale_length == length(CallTopics.locale_topics(locale))
```

Comment:
this doesn't test much, only that there are the same number of items in each locale, both can be 0